cilium/cmd/encrypt_status.go

```diff
@@ -142,3 +143,11 @@ func dumpIPsecStatus() {
 		}
 	}
 }
+
+func dumpWireGuardStatus(p *models.EncryptionStatus) {
+	for _, wg := range p.Wireguard.Interfaces {
+		fmt.Printf("Interface: %s\n", wg.Name)
+		fmt.Printf("\tPublic Key: %s\n", wg.PublicKey)
+		fmt.Printf("\tNo of Peers: %d", wg.PeerCount)
```

Comment:
Nit:
```suggestion
		fmt.Printf("\tNumber of peers: %d", wg.PeerCount)
```
`Key` in `Public Key` should also be lower case (i.e., `Public key:`).